docker/Dockerfile

```diff
@@ -83,15 +87,17 @@ RUN set -e; \
     esac; \
     case "$DISTRO_ID-$DISTRO_VERSION_ID" in \
     'debian-'* | 'linuxmint-'* | 'ubuntu-'*) \
-        curl --retry "$DEB_MAX_ATTEMPT" --retry-connrefused --retry-delay 3 -fsSL 'https://bazel.build/bazel-release.pub.gpg' \
-        | gpg --dearmor \
-        | sudo tee /etc/apt/trusted.gpg.d/bazel.gpg \
-        >/dev/null; \
-        echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" \
-        | sudo tee /etc/apt/sources.list.d/bazel.list; \
-        sudo apt-get autoremove -y; \
-        sudo apt-get clean; \
-        sudo rm -rf /var/lib/apt/lists/*; \
+        if [ "$TARGETARCH" == 'amd64' ]; then \
```

Comment:
Are you saying bazel doesn't support arm64?

Reply:
Bazel support arm64, but they don't have an arm64 deb source nor arm64 linux installer, but they do have a prebuilt binary.

> Note: For Arm-based systems, the APT repository does not contain an arm64 release, and there is no binary installer available. Either use Bazelisk or compile from source.

https://bazel.build/install/ubuntu

Reply:
Emmm....
Seems bazel doesn't even support bootstrapping.
https://docs.bazel.build/versions/5.2.0/install-compile-source.html#bootstrap-bazel

This is a big issue.
Pulling from GitHub within China inside a docker build simply doesn't work.

Reply:
Not sure if these will help:

- https://packages.debian.org/sid/main/bazel-bootstrap
- https://packages.ubuntu.com/search?keywords=bazel-bootstrap

Reply:
Also that is for compiling Bazel. We are downloading the prebuilt binary, does it still have the same issue?

Reply:
The point of compiling bazel is to avoid downloading prebuilt bin...